tests/core/OperationThroughputLimiter.spec.ts

```diff
@@ -0,0 +1,199 @@
+import AnchoredDataSerializer from '../../lib/core/versions/latest/AnchoredDataSerializer';
+import ITransactionStore from '../../lib/core/interfaces/ITransactionStore';
+import MockTransactionStore from '../mocks/MockTransactionStore';
+import ThroughputLimiter from '../../lib/core/versions/latest/ThroughputLimiter';
+import TransactionModel from '../../lib/common/models/TransactionModel';
+
+describe('OperationRateLimiter', () => {
+
+  let throughputLimiter: ThroughputLimiter;
+  let transactionStore: ITransactionStore;
+
+  function getTestTransactionsFor1Block () {
+    return [
+      {
+        transactionNumber: 1,
+        transactionTime: 1,
+        transactionTimeHash: 'some hash',
+        anchorString: AnchoredDataSerializer.serialize({
+          anchorFileHash: 'file_hash',
+          numberOfOperations: 12
+        }),
+        transactionFeePaid: 333,
+        normalizedTransactionFee: 1
+      },
+      {
+        transactionNumber: 2,
+        transactionTime: 1,
+        transactionTimeHash: 'some hash',
+        anchorString: AnchoredDataSerializer.serialize({
+          anchorFileHash: 'file_hash2',
+          numberOfOperations: 11
+        }),
+        transactionFeePaid: 998, // second highest fee should come second
+        normalizedTransactionFee: 1
+      },
+      {
+        transactionNumber: 3,
+        transactionTime: 1,
+        transactionTimeHash: 'some hash',
+        anchorString: AnchoredDataSerializer.serialize({
+          anchorFileHash: 'file_hash3',
+          numberOfOperations: 8
+        }),
+        transactionFeePaid: 999, // highest fee should come first
+        normalizedTransactionFee: 1
+      },
+      {
+        transactionNumber: 4,
+        transactionTime: 1,
+        transactionTimeHash: 'some hash',
+        anchorString: AnchoredDataSerializer.serialize({
+          anchorFileHash: 'file_hash4',
+          numberOfOperations: 1
+        }),
+        transactionFeePaid: 14,
+        normalizedTransactionFee: 1
+      }
+    ];
+  }
+
+  beforeEach(() => {
+    transactionStore = new MockTransactionStore();
+    throughputLimiter = new ThroughputLimiter(10, 25, transactionStore);
```

Comment:
Nit: For readability, can we assign const variables to 10 and 25 before passing so reader knows which is which.